corgidrp/l4_to_tda.py

```diff
@@ -1,5 +1,50 @@
 # A file that holds the functions that transmogrify l4 data to TDA (Technical Demo Analysis) data 
 import corgidrp.fluxcal as fluxcal
+import numpy as np
+
+def determine_app_mag(input_dataset, source_star):
```

Comment:
would it be useful to offer the ability to apply a scale factor to the source_star spectrum? I could see cases where a science target doesn't have a nicely created CALSPEC file, but we know it's spectral type is basically the same as a CALJSPEC source, but the fact that it's X times fainter. This could allow for a quick calibration without needing to generate a brand new CALSPEC file.

Reply:
Well, I could add scale_factor as optional parameter. At least it doesn't hurt.

Reply:
Looks good. thanks!